autocomplete_light/fields.py

```diff
@@ -88,13 +85,24 @@ class MultipleChoiceField(ChoiceField, forms.MultipleChoiceField):
 class ModelChoiceField(FieldBase, forms.ModelChoiceField):
     widget = ChoiceWidget
 
+    def __init__(self, autocomplete=None, registry=None, widget=None,
+            widget_js_attributes=None, autocomplete_js_attributes=None,
+            extra_context=None, *args, **kwargs):
+
+        super(ModelChoiceField, self).__init__(autocomplete, registry, widget,
+                widget_js_attributes, autocomplete_js_attributes,
+                extra_context, *args, **kwargs)
+
+        # Store ourself in the autocomplete, for AutocompleteModel.choices.
+        self.autocomplete.model_field = self
```

Comment:
Oh and be careful, it should be self.autocomplete.form_field here i guess, i'd assume self.autocomplete.model_field to be an instance of the fk/m2m model field.